dvc/commands/experiments/show.py

```diff
@@ -130,26 +135,28 @@ def _collect_rows(
         if not is_baseline:
             new_checkpoint = not (tip and tip == parent_tip)
 
-        row = [
-            exp_name,
-            name_rev,
-            typ,
-            _format_time(exp.get("timestamp"), fill_value, iso),
-            parent,
-            state,
-            executor,
-        ]
-        fill_value = FILL_VALUE_ERRORED if results.get("error") else fill_value
+        row_dict["Experiment"] = exp.get("name", "")
+        row_dict["rev"] = name_rev
+        row_dict["typ"] = typ
+        row_dict["Created"] = _format_time(
+            exp.get("timestamp"), fill_value, iso
+        )
+        row_dict["parent"] = parent
+        row_dict["State"] = state
+        row_dict["Executor"] = exp.get("executor", fill_value)
```

Comment:
Just a nitpicky suggestion, what do you think of using `row_dict.update()` here? I'm okay with how it is as well though.

Reply:
Do you mean like the following?

```suggestion
        row_dict.update(
            Experiment=exp.get("name", ""),
            rev=name_rev,
            typ=typ,
            Created=_format_time(
                exp.get("timestamp"), fill_value, iso
            ),
            parent=parent,
            State=state,
            Executor=exp.get("executor", fill_value)
        )
```

No strong preference, is `update` more efficient or is there another reason?

Reply:
@daavoo, I meant it more like following, to use dict arg than as kwargs:
```python
row_dict.update({
    "Experiment": exp.get("name", ""),
    "rev": name_rev,
     "typ": typ,
     "Created": _format_time(exp.get("timestamp"), fill_value, iso),
     "parent": parent,
     "State": state,
     "Executor": exp.get("executor", fill_value),
     ...
})
```

It's a matter of personal preference, I guess. As I said, I am fine as-is too.